viadot/tasks/sap_rfc.py

```diff
@@ -15,6 +16,7 @@ def __init__(
         self,
         query: str = None,
         sep: str = None,
+        replacement: str = "-",
```

Comment:
Not necessary to do that in `init()` because it is already explained in `run()`. However, if you think should be recommended do it in `init()`, I could do it.